lib/pbench/cli/agent/commands/results/__init__.py

```diff
@@ -58,42 +56,42 @@ def move_results(ctx, _user, _prefix, _show_server):
         logger.error("Failed to create temporary directory")
         sys.exit(1)
 
-    dirs = [
-        _dir
-        for _dir in next(os.walk(config.pbench_run))[1]
-        if not _dir.startswith("tools-") and not _dir.startswith("tmp")
-    ]
+    runs_copied = 0
+    failures = 0
```

Comment:
For the third time, I see no way here for `failures` to be incremented since you dropped checking the return value from `copy_result_tb()` ... what's the intent here? If we're really going to stop caring, let's not return it in the tuple rather than just saying `0`.

Reply:
See comments in #1947.